moveit_core/planning_scene/include/moveit/planning_scene/planning_scene.h

```diff
@@ -51,7 +51,6 @@
 #include <moveit_msgs/RobotTrajectory.h>
 #include <moveit_msgs/Constraints.h>
 #include <moveit_msgs/PlanningSceneComponents.h>
-#include <boost/enable_shared_from_this.hpp>
 #include <boost/noncopyable.hpp>
 #include <boost/shared_ptr.hpp>
```

Comment:
yes, but now thant you use the Ptr typedef they don't mention the symbol anymore. So this can be removed imo